jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/panel/PanelGroupGeneralTester.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.test.showcase.panel;
+
+import org.junit.Test;
+
+import com.liferay.faces.test.selenium.Browser;
+import com.liferay.faces.test.selenium.assertion.SeleniumAssert;
+import com.liferay.faces.test.showcase.TesterBase;
+
+
+/**
+ * @author  Kyle Stiemann
+ * @author  Philip White
+ */
+public class PanelGroupGeneralTester extends TesterBase {
+
+	@Test
+	public void runPanelGroupGeneralTest() throws Exception {
+
+		Browser browser = Browser.getInstance();
+		navigateToUseCase(browser, "panelGroup", "general");
+
+		// Test that the first example component renders a <span> and the second one a <div>.
+		SeleniumAssert.assertElementVisible(browser,
+			"//span[contains(@class,'-panel') and contains(text(),'This text is inside a <span> HTML tag')]");
```

Comment:
Please use `SeleniumAssert.assertElementTextVisible()` here as well. A (one-liner) method to factor out the common code here would also make this test more readable.